{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "psirt@us.ibm.com",
			"ID": "CVE-2023-30996"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "IBM Cognos Analytics 11.1.7, 11.2.4, and 12.0.0 could be vulnerable to information leakage due to unverified sources in messages sent between Windows objects of different origins.  IBM X-Force ID:  254290."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/254290",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/254290"
				},
				{
					"name": "https://security.netapp.com/advisory/ntap-20240405-0004/",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://security.netapp.com/advisory/ntap-20240405-0004/"
				},
				{
					"name": "https://security.netapp.com/advisory/ntap-20240621-0006/",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://security.netapp.com/advisory/ntap-20240621-0006/"
				},
				{
					"name": "https://www.ibm.com/support/pages/node/7123154",
					"tags": [
						"Vendor Advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/7123154"
				},
				{
					"name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/254290",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/254290"
				},
				{
					"name": "https://www.ibm.com/support/pages/node/7123154",
					"tags": [
						"Vendor Advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/7123154"
				},
				{
					"name": "https://security.netapp.com/advisory/ntap-20240621-0006/",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://security.netapp.com/advisory/ntap-20240621-0006/"
				},
				{
					"name": "https://security.netapp.com/advisory/ntap-20240405-0004/",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://security.netapp.com/advisory/ntap-20240405-0004/"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "NONE",
				"baseScore": 5.3,
				"baseSeverity": "MEDIUM",
				"confidentialityImpact": "LOW",
				"integrityImpact": "NONE",
				"privilegesRequired": "NONE",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
				"version": "3.1"
			},
			"exploitabilityScore": 3.9,
			"impactScore": 1.4
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:netapp:oncommand_insight:-:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			},
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack5:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "11.1.7",
						"versionStartIncluding": "11.1.1",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack6:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack7:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "11.2.4",
						"versionStartIncluding": "11.2.0",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.2.4:-:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack1:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.2.4:fixpack2:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:12.0.0:*:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:12.0.1:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-12-17T19:32Z",
	"publishedDate": "2024-02-26T16:27Z"
}
